Tika support (#998)
apache/camel-quarkus: extensions/tika/runtime/src/main/doc/limitations.adoc

Parameters `tikaConfig` and `tikaConfigUri` are not available in quarkus camel tika extension. Configuration
can be changed only via `application.properties`.

> While you can use any of the available Tika parsers in JVM mode (https://tika.apache.org/[Apache Tika]),
> only several Tika parses are supported in native mode. See https://quarkus.io/guides/tika[QUARKUS - USING APACHE TIKA].

Reviewer: ```suggestion
While you can use any of the available https://tika.apache.org/1.24.1/formats.html[Tika parsers] in JVM mode,
only some of those are supported in native mode - see the https://quarkus.io/guides/tika[Quarkus Tika guide].
```

In order to work in native mode, some properties should be set:

* `quarkus.tika.parsers`
* optionally `quarkus.tika.parser.*`

Example of `application.properties` follows :
[source,properties]
----
quarkus.tika.parsers = pdf,odf,office
quarkus.tika.parser.office = org.apache.tika.parser.microsoft.OfficeParser

> ----

Reviewer: My understanding is that using `quarkus.tika.parsers` is recommended (not required) to reduce the application memory and native executable sizes. So we should perhaps also formulate it like that.

It is not clear to me whether each abbreviation used in `quarkus.tika.parsers` needs to be defined in `quarkus.tika.parser.my-abbrev` ? Or are there some well known abbreviations?

> For more information about selecting parsers see https://quarkus.io/guides/tika[QUARKUS - USING APACHE TIKA]

Reviewer: ```suggestion
For more information about selecting parsers see https://quarkus.io/guides/tika[Quarkus Tika guide]
```
